src/LibVLCSharp.Uno/Themes/PlaybackControls.xaml

```diff
@@ -262,7 +262,7 @@
                                                                AutomationProperties.AccessibilityView="Raw">
                                                             <ToolTipService.ToolTip>
                                                                 <ToolTip x:Name="ThumbnailTooltip">
-                                                                    <ContentPresenter Content="{Binding}" />
+                                                                    <ContentControl Content="{Binding}" />
```

Comment:
Are you sure about this change?

Reply:
ok I give up I can't figure out where this tooltip should show up.  I have built straight 3.x on uno 2 as well and can't quite find what slider this references.    Is this just the position ellipse?  If so what should the tooltip be displaying?
![image](https://user-images.githubusercontent.com/1643324/122036801-d18f2e00-cd88-11eb-873c-0a6503f92d15.png)